src/courtroom.cpp

```diff
@@ -8,7 +8,7 @@ Courtroom::Courtroom(AOApplication *p_ao_app) : QMainWindow()
   qsrand(static_cast<uint>(QDateTime::currentMSecsSinceEpoch() / 1000));
 
   keepalive_timer = new QTimer(this);
-  keepalive_timer->start(60000);
+  keepalive_timer->start(1000);
```

Comment:
Why so frequent?

- Clock drift is not that extreme. For instance, NTP uses a polling interval ranging between 64 to 1024 seconds (based on the stability of the clock).
- Considering only a fringe of users will be using the timer functionality, and considering the timer is not actually being corrected with new ping packets, this will increase keepalive traffic for no apparent reason.